compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/IsolationAnalyzer.java

```diff
@@ -319,6 +319,8 @@ private void analyzeNode(BLangNode node, SymbolEnv env) {
     }
 
     public BLangPackage analyze(BLangPackage pkgNode) {
+        this.arrowFunctionTempSymbolMap.clear();
+        this.isolationInferenceInfoMap.clear();
```

Comment:
These are [already cleared](https://github.com/ballerina-platform/ballerina-lang/blob/274e1dca660da80cc2854bbba37cec60a9b51acb/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/IsolationAnalyzer.java#L3527) at the end of this method though, via [`inferIsolation`](https://github.com/ballerina-platform/ballerina-lang/blob/274e1dca660da80cc2854bbba37cec60a9b51acb/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/IsolationAnalyzer.java#L332).

Reply:
These are getting accumulate somehow when running in the LangServer mode. Need to check why.